WordPress/Classes/ViewRelated/Blog/Blog Details/Detail Header/ActionRow.swift

```diff
@@ -93,6 +94,13 @@ class ActionRow: UIStackView {
         spacing = Constants.minimumSpacing
         translatesAutoresizingMaskIntoConstraints = false
         refreshStackViewVisibility()
+
+        backgroundColor = .listForeground
+
+        layer.cornerRadius = 10
```

Comment:
I don't think we need to apply a corner radius anymore since the inset grouped style is rounding the corners of the Quick Actions cell

Reply:
Good catch! We also don't need the backgroundColor, removed both in 80a2a6bd238ba77a8c340db301a5dde5274cac1e